service splitServer{    
    string get_split_words(1:string text, 2:string language) 
}    
